### ISSUE TITLE
Host param info/text rescan doesn't rename params in alex-117053

### ISSUE THREAD

**Issue by baconpaul** [free-audio/interop-tracker#8]
If I call

```
_host.paramsRescan(CLAP_PARAM_RESCAN_INFO | CLAP_PARAM_RESCAN_TEXT);
```

on the main thread in BWS 4.3 beta 1 then the param info rescans but everything else does too. It seems to do a full restart of all audio ports.

This is way too much! I call this when I change a parameter name. So wanted to log the issue here.

Thanks!

**Comment by baconpaul** [comment]
@abique I upgraded surge so in the next nightly it will properly notify with PARAM_INFO | PARAM_VALUES | PARAM_TEXT

This means

1. If I change an oscillator type, bitwig shows me the correctly updated param names (great) but
2. Each time I change an oscillator type I get a *full plugin restart*

Log message from engine is

```
Restarting processing after parameters updated
Applying plugin IO state
PluginHost: Requesting restart of audio processing
Requesting restart of processing because IO buses or latency have changed
PluginDeviceContentsNode request restart processing
Starting fade out because request restart processing
Sending ready to restart processing
PluginDeviceContents ready to restart processing
PluginHost: Ready to restart audio processing
Ready to restart processing
Applying plugin instance state
```

running 116647

I think you need to restart if the flag is _ALL per the docs but not in the other cases. Note that the VST3 of surge doesn't engine restart in this case.

Thanks!

**Comment by baconpaul** [comment]
This problem still exists in 4.3 beta 3.

**Comment by abique** [comment]
Should be fixed in 4.3b4.

**Comment by baconpaul** [comment]
Hmm no now i don't get my parameters renaming at all. Let me look.

**Comment by baconpaul** [comment]
Alright in alex-117053 I see that sending

```
_host.paramsRescan(CLAP_PARAM_RESCAN_VALUES | CLAP_PARAM_RESCAN_INFO |
                                   CLAP_PARAM_RESCAN_TEXT);
```

no longer restarts my plugin (great!) but also doesn't rename my params (bummer).

It does put into the log each time 'plugin state applied' 

So I think this one still needs some tweaking.

**Comment by baconpaul** [comment]
Sorry if I revert to just calling with `_VALUES` and `_TEXT` it works in 43b3 with 0.25 but doesn't work in alex-117053 with 0.26

I merged a change so that 0.25 has just the pair.

**Comment by baconpaul** [comment]
OK and at 0.26 with just _V and _T in alex-117153 this works perfect.

So closing this issue!
